### ISSUE TITLE
I'm guessing this is abandoned?

### ISSUE THREAD

**Issue by ozra** [unn4m3d/cppize#5]
I worked on https://github.com/ozra/onyx-lang - using the Crystal AST quite vigorously until I got tied up in shitloads of work, and still am.

I've continued working on it "on paper", refining and reformulating ideas, and have since long decided "_when time makes itself available_" (_hehe, can dream_)  to rewrite Onyx from scratch, targeting C++ as "fundamental assembly", which was what I had actually intended from the beginning (_I've coded C++ since '98, just dislike the syntax, lack of modules, statements not being expressions, lack of union types and branch-inference, etc., bla bla bla bla_)

Anyway — somehow I never noticed this project before, which is a bummer, so I just had a lightbulb flick on: any thoughts on continuing it? :)

**Comment by unn4m3d** [comment]
Yep, this is abandoned :sweat: . 

This was my first (and only) transpiler writing experience, so I haven't thought about some architecture details, and one day I realized I need to redesign it and to rewrite almost from scratch. But when I started to think about rewriting, I gave up because it was too complex for me then.

Unfortunately, I don't have enough time to continue this project. Maybe I'll find some free time for it someday, but I'm not sure.
